_meta/beat.yml

```diff
@@ -79,6 +79,14 @@ output.elasticsearch:
       when.contains:
         processor.event: "sourcemap"
 
+    - index: "apm-%{[beat.version]}-%{+yyyy.MM.dd}-error"
```

Comment:
What do you think about doing 
`apm-%{[beat.version]}-error-%{+yyyy.MM.dd}`
or 
`apm-error-%{[beat.version]}-%{+yyyy.MM.dd}`
instead?

Having the date at the end seems more natural for me. It would mean that when sorting alphabetically, you'll get them grouped by version and data type first, and then date.